code/game/objects/items/storage/toolbox.dm

```diff
@@ -11,75 +11,61 @@
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb = list("robusted")
 
-
 /obj/item/storage/toolbox/emergency
 	name = "emergency toolbox"
 	icon_state = "red"
 	item_state = "toolbox_red"
-
-
-/obj/item/storage/toolbox/emergency/Initialize(mapload, ...)
-	. = ..()
-	new /obj/item/tool/crowbar/red(src)
-	new /obj/item/tool/extinguisher/mini(src)
-	if(prob(50))
-		new /obj/item/flashlight(src)
-	else
-		new /obj/item/flashlight/flare(src)
-	new /obj/item/radio(src)
-
+	spawns_with = list(
+		/obj/item/tool/crowbar/red,
+		/obj/item/tool/extinguisher/mini,
+		/obj/item/flashlight/flare,
+		/obj/item/radio,
+	)
```

Comment:
why are you not using your spawnprob system?

Reply:
it doesn't support if this else that